### ISSUE TITLE
Check whether the tag corresponds to the version in setup.py

### ISSUE THREAD

**Issue by vmarkovtsev** [rporres/python-package-server#13]
At https://github.com/src-d/pypi-on-github-indexer/blob/master/pypi_on_github_indexer/__main__.py#L85

We should add

```python
else if args.repo_tag.lstrip("v") != package_version:
    print("tag <> setup.py version mismatch: %s vs %s" % (args.repo_tag.lstrip("v"), package_version), file=sys.stderr)
    return 1
```

Otherwise it may lead to nasty version conflicts.

**Comment by rporres** [comment]
If we're going to enforce this convention, the `repo_tag` option does not have any use. That option was there to cover the case that a repo does not follow it.

The only reason to keep it in the way you propose is to verify that there are no version conflicts. In that case, it could be better to make the `repo_tag` option mandatory

**Comment by vmarkovtsev** [comment]
ok, changed to mandatory

**Comment by rporres** [comment]
Fixed by #14
